app/jobs/notify_stale_incident_channels.py

```diff
@@ -11,26 +11,33 @@ def notify_stale_incident_channels(client):
     channels = slack_channels.get_stale_channels(
         client, pattern=INCIDENT_CHANNELS_PATTERN
     )
-    text = """👋  Hi! There have been no updates in this incident channel for 14 days! Consider archiving it.\n
-        Bonjour! Il n'y a pas eu de mise à jour dans ce canal d'incident depuis 14 jours. Vous pouvez considérer l'archiver."""
+    text = """👋  Hi! There have been no updates in this incident channel for 14 days! Consider scheduling a retro or archiving it.\n
+        Bonjour! Il n'y a pas eu de mise à jour dans ce canal d'incident depuis 14 jours. Pensez à planifier une rétro ou à l'archiver."""
     attachments = [
         {
-            "text": "Would you like to archive the channel now? | Voulez-vous archiver ce canal maintenant?",
+            "text": "Would you like to archive the channel now or schedule a retro? | Souhaitez-vous archiver la chaîne maintenant ou programmer une rétro?",
```

Comment:
Would actually change the French here to:

Souhaitez-vous archiver le canal maintenant ou planifier une rétro?

Reply:
Done! Thank you so much for the correction.